phantombuster/web-browser-extension
# CP-422: Fix button removal without setup reload

## 📝 What is this PR

I found an issue when testing the [flow builder fix about editing the source](https://github.com/phantombuster/pb2-front/pull/3059) that comes from the extension : the connection button was not re-added in the field because there was no page reload and the interval was cleared

## 🏭 How does it work

the set interval loop continue forever but it the session cookie button is already there, there is a early return to prevent adding it again

## 🗒 Notes

I also set the `private _fastPoll = 100` to 200 because I think the impact for the user is mainly positive: 200ms is not perceptive and it might prevent some lags on old devices

## 🧪 QA Process

-   check there is no regression on agent setup (linkedin and facebook)
- check the bug is now fixed for the flow builder setup

## 🚀 What to do after deployment

I think we should cherry pick the fix on master for Firefox user if we are not able to deploy the manifest V3 on Firefox before the Flow Builder release
